frontend/src/InteractiveImport/Interactive/InteractiveImportRow.js

```diff
@@ -319,7 +319,9 @@ class InteractiveImportRow extends Component {
         >
           {
             showReleaseGroupPlaceholder ?
-              <InteractiveImportRowCellPlaceholder /> :
+              <InteractiveImportRowCellPlaceholder
+                className={styles.optional}
```

Comment:
Instead of passing in the `classname` and duplicating the CSS from `InteractiveImportRowCellPlaceholder.css` to `Interactive/InteractiveImportRow.css`, pass in a property of `isOptional`, which will default to true. If true it should either use a different class for the component or combine classes with `classNames()` (we do this in other components already).